src/lib.rs

```diff
@@ -406,24 +502,34 @@ where
         let page_lower = upper_left.0 as usize;
         let page_upper = lower_right.0 as usize;
 
-        buffer
+        for c in buffer
             .chunks(disp_width)
             .skip(starting_page)
             .take(num_pages)
             .map(|s| &s[page_lower..page_upper])
-            .try_for_each(|c| interface.send_data(U8(c)))
-    }
-
-    /// Release the contained interface.
-    pub fn release(self) -> DI {
```

Comment:
why would you remove his method?

Reply:
because it got lost while trying some different patterns :)